sumy/__main__.py

```diff
@@ -101,6 +102,7 @@ def handle_arguments(args, default_input_stream=sys.stdin):
     items_count = ItemsCount(args["--length"])
 
     language = args["--language"]
+    language = normalize_language(language)
```

Comment:
I am not sure this is enough. I would like to have the same API also for users of sumy that use it as library. Can you please normalize language on all places where language is consumed instead of here? I mean function `get_stop_words` will accept any format of language, the same as `Tokenizer` and `Stemmer`. That way also users using sumy as library will have benefit from your PR :)

Reply:
Done in ec5eb69